upgrade

```diff
@@ -30,82 +28,110 @@ if [ "$(whoami)" != "root" ]; then
   exit 3
 fi
 
-if [ $(printf "%s\n" "$SERVER_RELEASE" "$REQUIRED_RELEASE" | sort -V -r | head -1) = "$REQUIRED_RELEASE" ] ; then
-  echo "This upgrade script is meant to be run on existing v7.5 installations." >&2
-  echo "Please contact support for details on upgrading to v7.5." >&2
-  echo
-  exit 1
-fi
-
 # Use a lock file so failed scripts cannot be imediately retried
 # If not the backup created on this script would be replaced
 if ! mkdir /var/lock/lamassu-update; then
   echo "Script is locked because of a failure." >&2
   exit 1
 fi
 
-decho "stopping lamassu-server"
-supervisorctl stop lamassu-server >> ${LOG_FILE} 2>&1
-supervisorctl stop lamassu-admin-server >> ${LOG_FILE} 2>&1
-
-decho "unlinking ${NPM_BIN}/lamassu* old executables"
-find ${NPM_BIN} -type l \( -name "lamassu-*" -or -name "hkdf" \) -exec rm -fv {} \; >> ${LOG_FILE} 2>&1
-
-if [ -d "/usr/lib/node_modules/lamassu-server" ]; then
-    BKP_NAME=lamassu-server-$(date +%s)
-    decho "renaming old lamassu-server instance to ${BKP_NAME}"
-    mv -v "/usr/lib/node_modules/lamassu-server" "/usr/lib/node_modules/${BKP_NAME}" >> ${LOG_FILE} 2>&1
+if [[ "$UBUNTU_VERSION" == "16.04" ]]; then
+  echo "Detected Ubuntu version: 16.04. Executing upgrade script part 1 (you may need to reboot more than once to finish this step)..."
+
+  decho "updating to node 14"
+  npm install -g n
+  sudo n 14
+  sudo n which 14
+
+  decho "updating to ubuntu 18.04"
+  sudo apt-get update -y
+  sudo apt-get --with-new-pkgs upgrade -y
+  sudo apt-get dist-upgrade -y
+  sudo do-release-upgrade -f DistUpgradeViewNonInteractive
+
+  echo "Update part 1 complete! Reboot the server, make sure that Ubuntu is now on version 18.04 and run this script again to execute the second step."
```

Comment:
Part 1 of how many? How does an operator reboot the server? How does one make sure it's on 18.04 (and doesn't the script check that for them)?

Also, use spacing:

```
echo
echo "Update part 1 of 2 complete!"
echo "You must now reboot this server, log in again, and run this script again to complete the upgrade."
echo "Paired machines will not be online until you do so."
echo "Reboot your server by running the following command:"
echo
echo "shutdown -r now"
echo
```